tg/main.py

```diff
@@ -2,10 +2,10 @@
 import logging.handlers
 import signal
 import threading
-from curses import window, wrapper
+from curses import window, wrapper  # type: ignore
 from functools import partial
 
-from telegram.client import Telegram
+from telegram.client import Telegram  # type: ignore
```

Comment:
what kind of error these ignoring?
Maybe we can add it to mypy config?
Also, let's disable check for long lines, it's annoying. Black fixes them if it's possible.

Reply:
that was for missing import error
`../tg/main.py:8: error: Skipping analyzing 'telegram.client': found module but no type hints or library stubs`
ok, lets disable it with `--ignore-missing-imports` flag. 

Mypy doesn't check line lengths, that `noqa` was for my flake8, but let's remove that line for now